### ISSUE TITLE
Error when nodes name are string type and int type

### ISSUE THREAD

**Issue by DanialArmadilLo** [rafguns/linkpred#25]
```bash
def _sorted_tuple(t):
    a, b = t
    return (a, b) if a > b else (b, a)

TypeError: '>' not supported between instances of 'str' and 'int'
```

**Comment by rafguns** [comment]
Indeed, nodes are assumed to be all of the same type and sortable. Looking at this code again, I wonder if maybe we can just get rid of `_sorted_tuple` and do sth like this:

```python
def __eq__(self, other):
    try:
        return self.elements == other.elements
    except AttributeError:
        a, b = other
        return self.elements == (a, b) or self.elements == (b, a)
```

**Comment by rafguns** [comment]
Fixed in 58ff2e1.

I have kept `_sorted_tuple` for now: it ensures that the ranking of node pairs is always the same, even in case of ties.
